src/i18n/locale-en.json

```diff
@@ -458,8 +459,8 @@
     "security.new.password": "New password",
     "security.save.admin.settings": "Save administrator settings",
     "security.admin.pass.unset": "If you unset the password and then enable security, this administrator will not be able to log into GraphDB through the workbench. Are you sure that you want to continue?",
-    "resource.no_data": "No Data Available",
-    "resource.system.cpu_load.label": "System CPU Load",
+    "resource.no_data": "No data available",
```

Comment:
I can't remember. Did you put thje french labels for these new features?

Reply:
No, I wanted to get everything done and then add the labels. There is an issue for this.